src/task/index.html

```diff
@@ -106,6 +106,37 @@
             <option value="ASSIGNED">Assigned</option>
           </select>
         </div>
+        <div class="inputBox">
+          <label for="taskLevel">Task Level</label>
+          <br />
+          <div class="inputBox">
+            <label for="level" class="editable">
+              Category:<span class="required">*</span>
+            </label>
+            <select
+              required
+              class="input notEditing"
+              id="category"
+              name="category"
+            >
+              <option selected value="system_design">System Design</option>
+              <option value="frontend">Frontend</option>
+              <option value="backend">Backend</option>
```

Comment:
can we make all the letters uppercase in the category to maintain consistency?

Reply:
The labels are consistent.
We have the same in status also
![image](https://user-images.githubusercontent.com/78433013/196659488-9d8f23ec-dfed-4c51-ab27-eb5283aafa54.png)

Reply:
Are you concerned about the values ??

Reply:
yes, all the values of status are also stored in uppercase in our DB